src/radical/pilot/task_description.py

```diff
@@ -157,6 +165,11 @@ class TaskDescription(ru.TypedDict):
         specific functionality (pipes, I/O redirection) which cannot easily be
         mapped to other task attributes.
 
+        TASK_RAPTOR_MASTER and TASK_RAPTOR_WORKER are two special types of tasks
+        that define RAPTOR's engine master(s) and worker(s) components and their
```

Comment:
I don't think we use the term `engine` anywhere else in the context of Raptor, so would suggest to remove it.  Otherwise looks good to me - thanks!